vars/notifyBuildResult.txt

```diff
@@ -35,8 +35,6 @@ emails on Failed builds that are not pull request.
 * slackCredentials: What slack credentials to be used. Default value uses `jenkins-slack-integration-token`.
 * slackNotify: Whether to send or not the slack notifications, by default it sends notifications on Failed builds that are not pull request.
 * analyzeFlakey: Whether or not to add a comment in the PR with tests which have been detected as flakey. Default: `false`.
```

Comment:
Missing the `jobName`

Reply:
Whoops. Forgot to push that commit. Thanks!